DESIGN.md

```diff
@@ -41,8 +41,8 @@ Here is an example extension configuration `extensions.yml`:
 
 ```
 extensions:
-  - name: opensearch-sdk // extension name
-    uniqueId: opensearch-sdk-1 // identifier for the extension
+  - name: opensearch-sdk-java // extension name
```

Comment:
Can we please make this `sample-extension`?  Extension names are not and never should be this project's name. I have a request in #74 to change this, and changing it here will avoid creating a merge conflict that I'll need to handle. :-)

Reply:
fixed